### ISSUE TITLE
Error Detection 目标检测错误

### ISSUE THREAD

**Issue by littlecay** [IDEA-Research/GroundingDINO#331]
I use offlinemode, and     
    config_file = 'D:/deeplearning/Grounded-Segment-Anything-main/GroundingDINO/groundingdino/config/GroundingDINO_SwinT_OGC.py'  # change the path of the model config file
    checkpoint_path = 'D:/deeplearning/Grounded-Segment-Anything-main/bert-base-uncased/pytorch_model.bin'  # change the path of the model
    image_path = 'D:/deeplearning/Grounded-Segment-Anything-main/1.jpg'
    text_prompt = 'chair'
    output_dir = 'D:/deeplearning/Grounded-Segment-Anything-main/output'
like this, However, the output results seems random. New labels apprear.
出现了很多的乱码标签，且每次结果都是随机的
![pred](https://github.com/IDEA-Research/GroundingDINO/assets/1807012/84914b63-1dc1-4843-86dc-a6604d1036ff)
Anyone have any ideas? Thanks

**Comment by NielsRogge** [comment]
Hi,

See https://github.com/IDEA-Research/GroundingDINO/issues/321 for easy inference

**Comment by littlecay** [comment]
yeah, I've deal with this successfully, but still wondering why did this happens? Seems like caused by an abusing of wrong pth?
